opensir/models/sirx.py

```diff
@@ -37,15 +46,15 @@ class SIRX(Model):
     """ SIRX model definition """
 
     CSV_ROW = ["Days", "S", "I", "R", "X"]
-    NUM_PARAMS = 4
+    NUM_PARAMS = 5
```

Comment:
why is this 5? are there only 4 parameters? (alpha, betta, kappa_0, kappa)

Reply:
There is a parameter which needs to be fitted, i_0/x_0. See fit function.

Reply:
ah! I see it received another parameters